### PR TITLE
add cluster data

### PR DESCRIPTION
**Pull request by Daniel-GrunbergerCA**
add nodeAgent, storage enabled / disabled
add namespace

### CODE REVIEW

File: armometadata/armometadata.go [hunk 1]
```diff
@@ -22,6 +22,9 @@ type ClusterConfig struct {
 	GatewayRestURL            string `json:"gatewayRestURL"`            // in-cluster gateway component REST API url
 	KubevulnURL               string `json:"kubevulnURL"`               // in-cluster kubevuln component REST API url
 	KubescapeURL              string `json:"kubescapeURL"`              // in-cluster kubescape component REST API url
+	StorageEnabled            bool   `json:"storage"`                   // storage configuration (enabled/disabled)
+	NodeAgentEnabled          bool   `json:"nodeAgent"`                 // node agent configuration (enabled/disabled)
```
Comment (Bezbran) [inline armometadata/armometadata.go:26]: let's add all the features.
image scan
posture scan
otel collector

**Review by Bezbran — CHANGES_REQUESTED** [review]

File: armometadata/armometadata.go (hunk 1 above)
Comment (dwertent) [reply armometadata/armometadata.go:26]: @Daniel-GrunbergerCA

File: armometadata/armometadata.go [hunk 2]
```diff
@@ -22,6 +22,12 @@ type ClusterConfig struct {
...
+	Namespace                 string `json:"namespace"`                 // namespace to deploy the components
+	KubevulnEnabled           bool   `json:"kubevuln"`                  // kubevuln configuration (enabled/disabled)
+	KubescapeEnabled          bool   `json:"kubescape"`                 // kubescape configuration (enabled/disabled)
```
Comment (dwertent) [inline armometadata/armometadata.go:29]:
```suggestion
	PostureScanEnabled          bool   `json:"postureScanEnabled"`                 // kubescape configuration (enabled/disabled)
```

File: armometadata/armometadata.go [hunk 3]
```diff
@@ -22,6 +22,12 @@ type ClusterConfig struct {
 	GatewayRestURL            string `json:"gatewayRestURL"`            // in-cluster gateway component REST API url
 	KubevulnURL               string `json:"kubevulnURL"`               // in-cluster kubevuln component REST API url
 	KubescapeURL              string `json:"kubescapeURL"`              // in-cluster kubescape component REST API url
+	StorageEnabled            bool   `json:"storage"`                   // storage configuration (enabled/disabled)
+	NodeAgentEnabled          bool   `json:"nodeAgent"`                 // node agent configuration (enabled/disabled)
+	Namespace                 string `json:"namespace"`                 // namespace to deploy the components
+	KubevulnEnabled           bool   `json:"kubevuln"`                  // kubevuln configuration (enabled/disabled)
```
Comment (dwertent) [inline armometadata/armometadata.go:28]:
```suggestion
	ImageVulnerabilitiesScanningEnabled           bool   `json:"imageVulnerabilitiesScanningEnabled"`                  // kubevuln configuration (enabled/disabled)
```

File: armometadata/armometadata.go [hunk 4]
```diff
@@ -22,6 +22,12 @@ type ClusterConfig struct {
 	GatewayRestURL            string `json:"gatewayRestURL"`            // in-cluster gateway component REST API url
 	KubevulnURL               string `json:"kubevulnURL"`               // in-cluster kubevuln component REST API url
 	KubescapeURL              string `json:"kubescapeURL"`              // in-cluster kubescape component REST API url
+	StorageEnabled            bool   `json:"storage"`                   // storage configuration (enabled/disabled)
+	NodeAgentEnabled          bool   `json:"nodeAgent"`                 // node agent configuration (enabled/disabled)
```
Comment (dwertent) [inline armometadata/armometadata.go:26]:
```suggestion
	RelevantImageVulnerabilitiesEnabled          bool   `json:"relevantImageVulnerabilitiesEnabled"`                 // node agent configuration (enabled/disabled)
```

**Review by dwertent — CHANGES_REQUESTED** [review]
We should declare the feature and not the component.

**Review by dwertent — APPROVED** [review]

**Pull request merged**